pkg/liqoctl/status/utils/resources/resources_test.go

```diff
@@ -0,0 +1,125 @@
+		resExpected   corev1.ResourceList
+		cpuQuantity   = resource.NewScaledQuantity(1000, resource.Kilo)
+		memQuantity   = resource.NewQuantity(2000, resource.BinarySI)
+		podsQuantity  = resource.NewQuantity(1, resource.DecimalSI)
+		otherQuantity = resource.NewQuantity(4000, resource.DecimalSI)
+	)
+	const (
+		clusterID = "ID1"
+	)
+
+	BeforeEach(func() {
+		ctx = context.Background()
+		clientBuilder = *fake.NewClientBuilder().WithScheme(scheme.Scheme)
+		clientBuilder.WithObjects(
+			&sharev1alpha1.ResourceOffer{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "offerRemote",
+					Labels: map[string]string{
+						consts.ReplicationOriginLabel: clusterID,
+						consts.ReplicationStatusLabel: "true",
+					},
+				},
+				Spec: sharev1alpha1.ResourceOfferSpec{
+					ResourceQuota: corev1.ResourceQuotaSpec{
+						Hard: corev1.ResourceList{
+							corev1.ResourceCPU:    *cpuQuantity,
+							corev1.ResourceMemory: *memQuantity,
+							corev1.ResourcePods:   *podsQuantity,
+							"other":               *otherQuantity,
+						},
+					},
+				},
+			},
+			&sharev1alpha1.ResourceOffer{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "offerLocal",
+					Labels: map[string]string{
+						consts.ReplicationDestinationLabel: clusterID,
+						consts.ReplicationRequestedLabel:   "true",
+					},
+				},
+				Spec: sharev1alpha1.ResourceOfferSpec{
+					ResourceQuota: corev1.ResourceQuotaSpec{
+						Hard: corev1.ResourceList{
+							corev1.ResourceCPU:    *cpuQuantity,
+							corev1.ResourceMemory: *memQuantity,
+							corev1.ResourcePods:   *podsQuantity,
+							"other":               *otherQuantity,
+						},
+					},
+				},
+			},
+		)
+		cl = clientBuilder.Build()
+	})
+
+	JustBeforeEach(func() {
+		resExpected = corev1.ResourceList{
+			corev1.ResourceCPU:    *cpuQuantity,
+			corev1.ResourceMemory: *memQuantity,
+			corev1.ResourcePods:   *podsQuantity,
+			"other":               *otherQuantity,
+		}
+	})
+
+	When("Total Incoming resources are retrieved", func() {
+		It("Should return the correct resources", func() {
+			res, err := resources.GetIncomingTotal(ctx, cl, clusterID)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(resources.CPU(&res)).To(Equal(resources.CPU(&resExpected)))
+			Expect(resources.Memory(&res)).To(Equal(resources.Memory(&resExpected)))
+			Expect(resources.Pods(&res)).To(Equal(resources.Pods(&resExpected)))
+			Expect(resources.Others(&res)).To(Equal(resources.Others(&resExpected)))
+		})
+	})
+	When("Total Outgoing resources are retrieved", func() {
+		It("Should return the correct resources", func() {
+			res, err := resources.GetOutgoingTotal(ctx, cl, clusterID)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(resources.CPU(&res)).To(Equal(resources.CPU(&resExpected)))
+			Expect(resources.Memory(&res)).To(Equal(resources.Memory(&resExpected)))
+			Expect(resources.Pods(&res)).To(Equal(resources.Pods(&resExpected)))
+			Expect(resources.Others(&res)).To(Equal(resources.Others(&resExpected)))
+		})
+	})
```

Comment:
This test is somewhat weak, since local and remote resources are equal, and it would not make evident the case in which you are retrieving the wrong ones.